Library/Homebrew/tap.rb

```diff
@@ -1116,7 +1116,12 @@ def audit_exception(list, formula_or_cask, value = nil)
       return false unless list.include? formula_or_cask
       return list[formula_or_cask] if value.blank?
 
-      list[formula_or_cask] == value
+      case list[formula_or_cask]
+      when Array
+        list[formula_or_cask].include? value
+      when Hash
```

Comment:
```suggestion
      else
```
I think?

Reply:
I changed it to an `if` statement instead of `case`.